packages/lib-classifier/src/store/ClassificationStore.js

```diff
@@ -116,7 +116,9 @@ const ClassificationStore = types
           const annotation = classification.annotations.get(task.taskKey) || task.createAnnotation()
           // new annotations must be added to this store before we can modify them
           classification.annotations.put(annotation)
-          annotation.value = annotationValue
+          if (annotationValue) {
```

Comment:
Outside the scope of this PR, but maybe `annotation.update(annotationValue)` and then each annotation is responsible for validating the type and value of `annotationValue`.